com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -265,7 +265,7 @@ internal int GetMaxScreenSpaceShadows()
         /// <summary>
         /// Debug display settings.
         /// </summary>
-        public DebugDisplaySettings debugDisplaySettings { get { return m_DebugDisplaySettings; } }
+        public DebugDisplaySettings debugDisplaySettings { get { return m_CurrentDebugDisplaySettings; } }
```

Comment:
@JulienIgnace-Unity can you check this one? Is it one you have miss in the separate PR?

Reply:
Yeah, this needs to be reverted. As we discussed earlier, the proper fix was merged into staging a couple weeks ago.